src/VkColorPaletteManager.php

```diff
@@ -153,9 +153,9 @@ public static function inline_css() {
 				// 色はこのクラスでだけの利用なら直接指定でも良いが、他のクラス名で応用できるように一旦css変数に格納している.
 				$dynamic_css .= ':root{ --' . $color['slug'] . ':' . $color['color'] . '}';
 				// .has- だけだと負けるので :root は迂闊に消さないように注意
-				$dynamic_css .= ':root .has-' . $color['slug'] . '-color { color:var(--' . $color['slug'] . '); }';
-				$dynamic_css .= ':root .has-' . $color['slug'] . '-background-color { background-color:var(--' . $color['slug'] . '); }';
-				$dynamic_css .= ':root .has-' . $color['slug'] . '-border-color { border-color:var(--' . $color['slug'] . '); }';
+				$dynamic_css .= ':root .has-' . $color['slug'] . '-color { color:var(--' . $color['slug'] . ') !important; }';
+				$dynamic_css .= ':root .has-' . $color['slug'] . '-background-color { background-color:var(--' . $color['slug'] . ') !important; }';
+				$dynamic_css .= ':root .has-' . $color['slug'] . '-border-color { border-color:var(--' . $color['slug'] . ') !important; }';
```

Comment:
```suggestion
				$dynamic_css .= '.has-' . $color['slug'] . '-color { color:var(--' . $color['slug'] . ') !important; }';
				$dynamic_css .= '.has-' . $color['slug'] . '-background-color { background-color:var(--' . $color['slug'] . ') !important; }';
				$dynamic_css .= '.has-' . $color['slug'] . '-border-color { border-color:var(--' . $color['slug'] . ') !important; }';
```

@shimotmk !important つけるなら :root は消しちゃっても良いかなと思います。

が、これ

// .has- だけだと負けるので :root は迂闊に消さないように注意

ってかいてあるけど、どのケースで負けるのか書いてないのが良くないですね...。

でも 「:root あり !important 無し」よりも 「:root なし !important あり」の方が強いので、 :root 消しちゃいましょう。上のコメントとともに。

Reply:
@kurudrive 
Good catch
 :rootとコメント削除しておきました！